syncing/src/main/kotlin/maru/syncing/pipeline/BeaconChainDownloadPipelineFactory.kt

```diff
@@ -23,7 +25,9 @@ class BeaconChainDownloadPipelineFactory(
   private val metricsSystem: MetricsSystem,
   private val peerLookup: PeerLookup,
   private val downloaderParallelism: Int,
```

Comment:
Usually the pattern we use to configure classes is to define a `data class Config`, which makes it easier to instantiate and refactor later on. In the client code it would be referred to as `BeaconChainDownloadPipelineFactory.Config`

Reply:
done